_posts/2017-02-17-LPL-SHOP.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: page
+title: "The Lucy Parsons Labs Online Store"
+#
+# Content
+#
+author: brian_kroll
+teaser: "Official Lucy Parsons Labs merch is hitting the streets through a new on-line store to help fund projects and cover costs."
+categories:
+   - website
+tags:
+   - website
+   - funding
+   - store
+   - posters
+   - stickers
+   - swag
+#
+# Styling
+#
+header: no
+image:
+ thumb: "thumbs/swag.png"
+#  header: ""
+#  thumb: ""
+#  homepage: ""
+#  caption: ""
+#  url: ""
+---
+Starting today, official Lucy Parsons Labs merchandise is available on [our new on-line shop](https://lucyparsonslabs.bigcartel.com/)! All proceeds go directly to funding our projects and help cover costs like web hosting and legal fees. We have two items for sale, the first is a great poster that is printed on heavy poster stock with great color contrasts and printed with union labour right here in Chicago! The second item is a great sticker pack designed by one of our awesome volunteers! The stickers are printed on heavy outdoor vinyl and look great on laptops or cop cars!
```

Comment:
Sure will change to "We have two items for sale, the first is a poster that is printed on heavy poster stock with great color contrasts and printed by a union shop right here in Chicago."